pkg/coderefs/coderefs.go

```diff
@@ -24,13 +25,12 @@ import (
 // PUTing a massive json payload. These limits will likely be tweaked over
 // time. The LaunchDarkly backend will also apply limits.
 const (
-	minFlagKeyLen                     = 3
-	maxFileCount                      = 5000
-	maxLineCharCount                  = 500
-	maxHunkCount                      = 5000
-	maxHunksPerFileCount              = 1000
-	maxHunkedLinesPerFileAndFlagCount = 500
-	maxProjKeyLength                  = 20
+	minFlagKeyLen                     = 3     // Minimum flag key length helps reduce the number of false positives
+	maxFileCount                      = 10000 // Maximum number of files containing code references
+	maxLineCharCount                  = 500   // Maximum number of characters per line
+	maxHunkCount                      = 25000 // Maximum number of total code references
+	maxHunkedLinesPerFileAndFlagCount = 500   // Maximum number of lines per flag in a file
+	maxProjKeyLength                  = 20    // Maximum project key length
```

Comment:
Increased limits for expected increase in number of references - these might have been too restrictive in the first place